src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.hpp

```diff
@@ -75,8 +75,12 @@ static_assert(OPENTHREAD_API_VERSION >= 219, "OpenThread version too old");
 // Network commissioning
 namespace {
 #ifndef _NO_NETWORK_COMMISSIONING_DRIVER_
+#ifndef CHIP_DEVICE_CONFIG_THREAD_NETWORK_ENDPOINT_ID
+#define CHIP_DEVICE_CONFIG_THREAD_NETWORK_ENDPOINT_ID (0)
+#endif
```

Comment:
This should be in a config header (either one that includes some sort of consumer-provided config, or one generated based on gn args).

In particular, it should be _very_ clear to someone using the SDK how to set this properly; right now the only "proper" way to do it seems to be "ensure it's in the command line when this file is being compiled or something".

Reply:
Added a gn arg to define the endpoint id.